orchestration/task_dispatcher.py

```
@@ -982,64 +1124,136 @@ def create_dynamic_agent(self, agent_spec: dict) -> bool:
 # Keep session alive for 1 hour for monitoring and debugging
-echo "[$(date)] Agent execution completed. Session remains active for monitoring."
-echo "[$(date)] Session will auto-close in 1 hour. Check log at: {log_file}"
-echo "[$(date)] Monitor with: tmux attach -t {agent_name}"
+echo "[$(date)] Agent execution completed. Session remains active for monitoring." | tee -a {log_file_quoted}
+echo "[$(date)] Session will auto-close in 1 hour. Check log at: {log_file_display}" | tee -a {log_file_quoted}
+echo "[$(date)] Monitor with: tmux attach -t {monitor_hint}" | tee -a {log_file_quoted}
 sleep {AGENT_SESSION_TIMEOUT_SECONDS}
 '''
 
-            script_path = Path("/tmp") / f"{agent_name}_run.sh"
+            script_path = Path("/tmp") / f"{agent_token}_run.sh"
```

Ruff (0.13.3)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)